packages/edit-site/src/components/editor/index.js

```diff
@@ -165,12 +165,11 @@ export default function Editor( { isLoading } ) {
 
 	let title;
 	if ( hasLoadedPost ) {
-		const type = typeLabels[ editedPostType ] ?? __( 'Template' );
 		title = sprintf(
 			// translators: A breadcrumb trail in browser tab. %1$s: title of template being edited, %2$s: type of template (Template or Template Part).
 			__( '%1$s ‹ %2$s ‹ Editor' ),
 			getTitle(),
-			type
+			typeLabels[ editedPostType ] ?? typeLabels.wp_template
```

Comment:
Unrelated change; I hope it's okay to include it here. I just simplified things a bit.

Reply:
I don't have any issues with the change. It could be a personal preference for readability but simple is good.